### DISCUSSION TITLE
Error while using workshop version

### DISCUSSION THREAD

**Discussion by Vysakh13579** [phoebe-project/phoebe2#527] [Questions]
To interpret convergence in MCMC as per 2021 workshop tutorials, I installed the workshop version using the script given in the workshop materials. After building wheels, the install returned a few errors, mentioning compatibility issues of a few modules like TensorFlow and scipy with numpy. I corrected this by installing the specified version. 

But after the install, when I ran the test script given on the workshop page, it returned an error message saying "`ValueError: cannot convert  to solar.`" This happened during the execution of the run_solver function. After looking at the source code, I found that this happened while PHOEBE(workshop version) attempted to convert the astropy units to PHOEBE units. Is this because of the version of astropy I am using? 
Could anyone please help me rectify this? 

![errorMSG](https://user-images.githubusercontent.com/72174961/133553712-cf46c7ff-9914-4ef4-8afb-de80447e1bca.png)

**Comment by kecnry** [comment]
This was an error introduced by a change in a recent astropy update and should be fixed in the 2.3.53 release, but the workshop version is not kept up-to-date with new fixes.  So you have a few options:

* downgrade astropy and continue using the workshop branch.  But be aware that some features were in the "preview" state in the workshop version and have not yet been released and you will be missing other minor bugfixes.
* use the latest 2.3.53 release which includes a fix for this (although there are some remaining issues with the latest versions of astropy - see #526).  This way you'll have the latest bugfixes, but you will be without some of those preview features mentioned in the workshop.
* use the lateset release and the [official tutorials](http://phoebe-project.org/docs/latest/tutorials) instead of the archived workshop content.

**Reply by Vysakh13579** [reply]
Okay, thanks for the help.
